Header header
string full_data
int32 Satellite_Num
float32 DOP_info
string Imu_aligned
string GPS_aligned
float64 latitude
float64 longitude
float64 altitude
float32 north_v
float32 east_v
float32 down_v
float32 total_v
float64 roll
float64 pitch
float64 heading
float64 track_angle
float32 angle_rate_x
float32 angle_rate_y
float32 angle_rate_z
float32 longitudinal_a
float32 traverse_a
float32 down_a